docs/user-guide/tasks/http.md

```diff
@@ -116,16 +123,16 @@ In this example, all three endpoints will have a `numRequests` of 50.
 
 The results of the `http` task is visualized using the `http` Iter8 Grafana dashboard. The dashboard can be found [here](https://raw.githubusercontent.com/iter8-tools/iter8/v0.16.2/grafana/http.json).
 
-To use the dashboard:
+Assuming the URL to the Grafana service is `$GRAFANA_URL`, you can install the dashboard as follows:
 
 1. Open Grafana in a browser. 
 2. Add a new data JSON API data source with the following parameters
-    * URL: `<link to Grafana service>/httpDashboard`
-    * Query string: `namespace=<namespace of experiment>&experiment=<name of experiment>`
+    * URL: `$GRAFANA_URL/grpcDashboard`
+    * Query string: `namespace=<namespace>&experiment=<test name>`
```

Comment:
```suggestion
    * Query string: `namespace=<namespace>&test=<test name>`
```

Reply:
Will wait for a subsequent PR (and changes to Iter8) for this change. Created issue https://github.com/iter8-tools/iter8/issues/1620. The current change applies to v0.16 and and can be merged now. The proposed change is an additional change for v0.17.